cmd/stack_create_test.go

```diff
@@ -196,16 +221,26 @@ func TestStackCreateInvalidLongStackName(t *testing.T) {
 	sandbox, cleanup := cmdtest.TestSetupWithSandbox(t, true)
 	defer cleanup()
 
-	args := []string{"stack", "create", "testing_stacktesting-stacktesting-stacktesting-stacktesting-stacktesting-stacktesting-stacktesting-stacktesting-stacktesting-stack"}
+	testStackName := "testing_stacktesting-stacktesting-stacktesting-stacktesting-stacktesting-stacktesting-stacktesting-stacktesting-stacktesting-stack"
+
+	args := []string{"stack", "create", testStackName}
 	_, err := cmdtest.RunAppsody(sandbox, args...)
 
 	if err == nil {
 		t.Fatal(err)
```

Comment:
A better message here would be nice as to why the test is failing. The `err` will be `nil` at this point.